packages/forma-36-website/src/pages/components/workbench/index.mdx

```diff
@@ -5,3 +5,5 @@ status: 'alpha'
 github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/Workbench'
 storybook: 'https://f36-storybook.contentful.com/?path=/story/alpha-workbench--default'
 ---
+
+Detailed documentation will be provided soon. For now, please have a look into the storybook for implementation details.
```

Comment:
@burakukula I added this thing here, hope you don't mind 😄

Reply:
@gui-santos thx a lot! 🎉